tests/create_katsu_test_datasets.py

```diff
@@ -0,0 +1,347 @@
+import requests
+import io
+
+
+KATSU_URL = "http://localhost:8001"
+
+
+def post_data(url, data):
+    response = requests.post(url, json=data)
+    print(response.json())
+    return response.json()
+
+
+'''
+does a post request to /api/projects
+creates a project with the project title passed in
+returns the project id
+'''
```

Comment:
Looks good - let's put these comments as docstrings within the function:

```
def get_project_id(project_title):
    '''
    does a post request....
    '''
```

That way if someone does e.g. `help(get_project_id)` they'll get the doctoring